tests/dummy/app/locales/en/translations.js

```diff
@@ -440,102 +440,108 @@ Ember.$.extend(true, translations, {
       },
       'flexberry-field': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-field/settings-example'
+          'caption': 'Flexberry-field. Settings example'
         }
       },
       'flexberry-file': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-file/settings-example'
+          'caption': 'Flexberry-file. Settings example'
         }
       },
       'flexberry-groupedit': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-groupedit/settings-example'
+          'caption': 'Flexberry-groupedit. Settings example'
         },
         'model-update-example': {
-          'caption': 'Components-examples/flexberry-groupedit/model-update-example',
+          'caption': 'Model update example',
           'addDetailButton': 'Add detail',
           'removeDetailButton': 'Remove detail',
         }
       },
       'flexberry-lookup': {
         'settings-example': {
-          'caption': 'Components-examples/flexberry-lookup/settings-example'
+          'caption': 'Flexberry-lookup. Settings example'
         },
         'customizing-window-example': {
-          'caption': 'Components-examples/flexberry-lookup/customizing-window-example'
+          'caption': 'Window customization',
+          'titleLookup': 'Master'
         },
         'limit-function-example': {
-          'caption': 'Components-examples/flexberry-lookup/limit-function-example'
+          'caption': 'Flexberry-lookup. Limit function example',
+          'titleLookup': 'Master'
         },
         'lookup-block-form-example': {
-          'caption': 'Components-examples/flexberry-lookup/lookup-block-form-example',
+          'caption': 'Lookup block form example',
```

Comment:
I think it should be named as "Flexberry-lookup. Lookup block form example", i.e. uniformly with other captions.
The same applies to the all other captions including russian translations.